kernel/include/kernel/vm.h

```diff
@@ -46,6 +46,7 @@ __BEGIN_CDECLS
 
 #define PAGE_ALIGN(x) ALIGN(x, PAGE_SIZE)
 #define IS_PAGE_ALIGNED(x) IS_ALIGNED(x, PAGE_SIZE)
+extern struct list_node arena_list;
```

Comment:
Would be a little cleaner to go ahead and leave this unexported and add a new simple function that returns a pointer to the arena list.

Reply:
Done